index.js

```diff
@@ -24,7 +24,8 @@ app.get(BASE_API_PATH + "/contacts", (req,res)=>{
     console.log(Date() + " - GET /contacts");
     db.find({}, (err,contacts)=>{
         if(err){
-            console.log(Date() + "  - " + err)
+            console.log(Date() + "  - " + err);
+            res.sendStatus(500);
         }else{
             res.send(contacts.map((contact)=>{
```

Comment:
Please, change the code to remove the _id from the information that is sent to the user.

Reply:
Changes done!